packages/app/src/main.ts

```diff
@@ -19,7 +19,15 @@ window.__vite__ = true
 
 const app = createApp(App)
 
-const ws = createWebsocket()
+function decodeBase64Unicode (str: string) {
```

Comment:
Updated here: https://github.com/cypress-io/cypress/commit/5f1fa1b0e142b81d93a184d93048b48ab5f4e53b

I renamed the file to base64.ts since it both encodes and decodes